### ISSUE TITLE
Ability to search videos within the specific channel

### ISSUE THREAD

**Issue by skyGtm** [TeamNewPipe/NewPipe#5441]
### Checklist

- [x] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [x] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [x] This issue contains only one feature request. I will open one issue for every feature I want to request.

#### Describe the feature you want

Ability to search videos within the specific channel.
I currently, for above mentioned task, use web version of Youtube, search the video on the desired Youtube Channel and copy link, paste url in NewPipe and watch; which I think is tedious task.

#### Is your feature request related to a problem? Please describe it

Generally, I prefer the videos on some topics from my favorite youtubers only. So, it is tedious to just scroll down the videos list to search videos on specific topic.
And using search 'keyword + channel name' (Example: router LinusTechTips) don't always provides desired results.

#### Additional context

There is similar issue, which is for subscription list only #4795
Screenshot of web version of Youtube with channel specific search.
![Capture](https://user-images.githubusercontent.com/50160188/104844225-16f12180-58f7-11eb-9576-a15a00e23a7d.PNG)

#### How will you/everyone benefit from this feature?

I think, it will help most of users from searching videos on specific topic from their favorite youtubers

**Comment by mhmdanas** [comment]
Related: #110.

**Comment by shivasagarrao** [comment]
Duplicate of [#2414](https://github.com/TeamNewPipe/NewPipe/issues/2414)

**Comment by mhmdanas** [comment]
@shivasagarrao you're not wrong, but I'd prefer to keep this one open, as that one requests multiple features.

**Comment by arncor** [comment]
I would love to see that feature as well ;)

**Comment by ElenaXX** [comment]
I would like this feature too !!! A program like newpipe can't miss it. Trashing requests as duplicates or such things is just an excuse when one doesn't want to do them. Just code it, is much faster than searching for alleged duplicates 😡

**Comment by Ronkn** [comment]
The supposed duplicate linked above for channel tabs is closed and implemented. This specific feature is still not present. Hoping it makes it in someday. I do exactly what the guy above does. It is in fact annoying.
